File: Exemplos/App_eXperience_Java/app/src/main/java/com/elgin/elginexperience/shipay/models/Buyer.java
package com.elgin.elginexperience.shipay.models;

import com.google.gson.annotations.SerializedName;

public class Buyer {
    public Buyer(String first_name, String last_name, String cpf, String email, String phone) {
        this.first_name = first_name;
        this.last_name = last_name;
        this.cpf = cpf;
        this.email = email;
        this.phone = phone;
    }

    @SerializedName("first_name")
    public String first_name;
    @SerializedName("last_name")
    public String last_name;
    @SerializedName("cpf")
    public String cpf;
    @SerializedName("email")
    public String email;
    @SerializedName("phone")
    public String phone;
}

File: Exemplos/App_eXperience_Kotlin/app/src/main/java/com/example/e1_elgin_kotlin/shipay/models/Buyer.kt
package com.example.e1_elgin_kotlin.shipay.models

import com.google.gson.annotations.SerializedName

data class Buyer(
    @field:SerializedName("first_name") var first_name: String,
    @field:SerializedName("last_name") var last_name: String,
    @field:SerializedName("cpf") var cpf: String,
    @field:SerializedName("email") var email: String,
    @field:SerializedName("phone") var phone: String
)
